src/healthcheck/mod.rs

```diff
@@ -2,10 +2,11 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, you can obtain one at https://mozilla.org/MPL/2.0/.
 
-//! Route handlers for our heathcheck endpoints:
-//! for the `GET /__version__` endpoint,
-//! for the `GET /__lbheartbeat__` endpoint and
-//! for the `GET /__heartbeat__` endpoint,
+//! Route handlers for our heathcheck endpoints.
+//!
+//! * `GET /__version__`
+//! * `GET /__lbheartbeat__`
+//! * `GET /__heartbeat__`
```

Comment:
Opted to tweak this while I was in here, because it looked a bit busy in the generated docs with them all on the same line:

<img width="829" alt="Screenshot of the generated documentation for the healthcheck module" src="https://user-images.githubusercontent.com/64367/46409016-f8f25980-c70b-11e8-9163-98a927ab0bdc.png" />